fluent-react/src/provider.ts

```diff
@@ -29,14 +28,7 @@ export function LocalizationProvider(
 ): ReactElement {
   return createElement(
     FluentContext.Provider,
-    {
-      value: props.l10n
-    },
+    { value: props.l10n },
     props.children
   );
 }
-
-LocalizationProvider.propTypes = {
```

Comment:
Question: Is this a behavior change for our users? Not everyone uses TypeScript, and this could provide valuable runtime checks if users are relying on the added safety here.

Reply:
The `prop-types` methods are effectively no-ops in production, so I don't think that this should be considered a change in behaviour. It's removed here because issues like facebook/prop-types#354 make it difficult to support `react@17` while keeping it as a dependency.

Reply:
I would somewhat disagree with the change in behavior, as it's still used in development mode. However, I won't block on this change if you feel the dependency management is too complex.